src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/HResults.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    internal static class HResults
+    {
+        public const int S_OK = 0;
+        public const int S_FALSE = 1;
+        public const int E_FAIL = -1;
```

Comment:
This is grounds for punishment... HRESULT aren't arbitrary and can't redefined to any value. This has a value and it should be respected - `0x80004005`.

Reply:
I thought I had that one right.  I'll fix that.

Reply:
Guess my memory was off.